nwb_conversion_tools/baselfpextractorinterface.py

```diff
@@ -69,8 +68,9 @@ def run_conversion(
             nwbfile=nwbfile,
             metadata=metadata,
             use_times=use_times,
-            write_as_lfp=True,
+            write_as='lfp',
+            es_key='LFPElectricalSeries',
```

Comment:
This default value for `es_key` does not align with your proposed default name on the spikeextractors PR

Reply:
this is specific for your baselfpinterface, it will overwrite the spikeinterface defaults

Reply:
I see now, hopefully that should work then